### PR TITLE
[BugFix][Excel] Fix read formulas/number cell value of excel

### PR DESCRIPTION
**Pull request by Tu-maimes**
…ation of data accuracy for the number data type

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Tu-maimes** [comment]
img_v3_02hl_09672128-e1b6-410d-9a70-397c5983122g

**Comment by hailin0** [comment]
Please check ci errors & add testcase

File: seatunnel-connectors-v2/connector-file/connector-file-base/src/test/java/org/apache/seatunnel/connectors/seatunnel/file/writer/ExcelReadStrategyTest.java [hunk 1]
```diff
@@ -227,6 +227,75 @@ public void testExcelReadDateString() throws IOException, URISyntaxException {
         }
     }
 
+    @Test
+    public void testExcelReadFORMULA() throws IOException, URISyntaxException {
```
Comment (hailin0) [inline seatunnel-connectors-v2/connector-file/connector-file-base/src/test/java/org/apache/seatunnel/connectors/seatunnel/file/writer/ExcelReadStrategyTest.java]:
```suggestion
    public void testExcelReadFormula() throws IOException, URISyntaxException {
```

File: seatunnel-connectors-v2/connector-file/connector-file-base/src/test/java/org/apache/seatunnel/connectors/seatunnel/file/writer/ExcelReadStrategyTest.java [hunk 2]
```diff
@@ -227,6 +227,75 @@ public void testExcelReadDateString() throws IOException, URISyntaxException {
...
+        URL excelFile =
+                ExcelReadStrategyTest.class.getResource("/excel/test_read_excel_FORMULA.xlsx");
```
Comment (hailin0) [inline seatunnel-connectors-v2/connector-file/connector-file-base/src/test/java/org/apache/seatunnel/connectors/seatunnel/file/writer/ExcelReadStrategyTest.java]:
```suggestion
                ExcelReadStrategyTest.class.getResource("/excel/test_read_excel_Formula.xlsx");
```

**Review by hailin0 — APPROVED** [review]
LGTM

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**